### PR TITLE
[homework#10]-js-part-5 by Ilya Ischenko

### PR DESCRIPTION
**Pull request by ilya07092000**

### CODE REVIEW

File: homeworks/ilya-ischenko/10-js-part-5/tasks/script.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+}
+console.log(digitCheck('1fsdfsdf'));
+console.log(digitCheck('fs1dfsdf'));
+
+// TASK 7 OPTIONAL Check if this entry is a phone number
+// Ukraine: +380 xx-xxx-xxxx
+// starts with 380 and after that 9 digits
+function checkPhoneNum(num) {
+  const number = num.replace(/\D+/g, '');
+  if (number.match(/380\d{9}/)) {
```
Comment (esvyridov) [inline homeworks/ilya-ischenko/10-js-part-5/tasks/script.js:69]: could you please refactor that so it checks only this `+380 xx-xxx-xxxx` format (width all spaces and dashes)

File: homeworks/ilya-ischenko/10-js-part-5/tasks/script.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  if (typeof msg !== 'string') {
+    throw TypeError(`Message should be of type string but was of type ${typeof msg}!`);
+  }
+
+  if (msg.length > 255 || msg.length === 0) {
+    throw RangeError(`Message contains ${msg.length} characters!`);
+  }
+
+  /*
+    сработала только проверка на всё кроме цифр,
```
Comment (esvyridov) [inline homeworks/ilya-ischenko/10-js-part-5/tasks/script.js:16]: could you please describe the issue or provide test cases?

Comment (ilya07092000) [reply homeworks/ilya-ischenko/10-js-part-5/tasks/script.js:16]: image

Comment (ilya07092000) [reply homeworks/ilya-ischenko/10-js-part-5/tasks/script.js:16]: So, I try to check HTML tags with regexp, that means "one an more word OR zero and more spaces" but it does not work

Comment (ilya07092000) [reply homeworks/ilya-ischenko/10-js-part-5/tasks/script.js:16]: Maybe it's problems with kata's tests, because it worked it in my ide

**Review by esvyridov — APPROVED** [review]

**Pull request merged**